utils.go

```diff
@@ -197,9 +198,9 @@ func parseMsg(msg *pb.WebcastResponse_Message, warnHandler func(...interface{}),
 						ID:       int64(user.Id),
 						Username: user.ProfileId,
 						Nickname: user.Name,
-						ProfilePicture: &ProfilePicture{
-							Urls: urls,
-						},
+						// ProfilePicture: &ProfilePicture{
```

Comment:
Not sure why you commented this out.  You should be able to remove the range for urls array above and just use `users = append(users, toUser(user))` here.